apps/src/blockly/addons/cdoWorkspaceSvg.js

```diff
@@ -52,8 +91,12 @@ export default class WorkspaceSvg extends GoogleBlockly.WorkspaceSvg {
     return Blockly.mainBlockSpace.getMetrics().toolboxWidth;
   }
 
-  // Use visibility:hidden not display:none to hide the trashcan so that it still takes up space, which is important
```

Comment:
I'm a little concerned about this, but unfortunately I don't remember what issues with the lid opening I was referring to here 😬 
From manual testing, I can't see any issues with the trashcan lid, and this comment is from ~1 year ago [ref](https://github.com/code-dot-org/code-dot-org/commit/4f15fe44ca5e3535fe928cb9fc263d2cd949d2a7) at which point we were on Blockly `3.20200625.2`. Currently we're on Blockly `6.20210701.0` so it's very possible/likely that things have changed enough in Blockly core that this is no longer relevant.